contrib/action_recognition/r2p1d/environment.yml

```diff
@@ -0,0 +1,30 @@
+name: r2p1d
```

Comment:
General comments:

- Two of the notebooks start with "r2p1d". That's not needed, since the subfolder has that name anyway
- The video_transformation.ipynb notebook has a few warnings in them. Should they be quick to fix, then why not
- Is there any reason for the folders "cv/utils" and "vu/models". Why not just put all .py scripts under "vu" directly and then delete those two folders?

Reply:
* Refactored the notebook names to be the same convention as our core modules.
* Let me fix that later (or fix them when we move to the core modules) as I really don't have much time now :-(
* For that, I followed torch's convention, but we can always change that later. I would say better time for refactoring will be when we merge this project into the core modules.